main.nf

```diff
@@ -625,6 +628,7 @@ if(params.aligner == 'hisat2'){
         script:
         index_base = hs2_indices[0].toString() - ~/.\d.ht2/
         prefix = reads[0].toString() - ~/(_R1)?(_trimmed)?(_val_1)?(\.fq)?(\.fastq)?(\.gz)?$/
+        def seqCenter = params.seqCenter ? "--rg-id ${prefix} --rg CN:${params.seqCenter.replaceAll("\\s","_")}" : ''
```

Comment:
This is now triggering a syntax error in the [Travis CI tests](https://travis-ci.org/nf-core/rnaseq/jobs/415394188):

```
_nf_script_6da76d62: 631: Variable `prefix` already defined in the process scope @ line 631, column 54.
    params.seqCenter ? "--rg-id ${prefix} -
                                 ^
```

Reply:
Also, I don't think that you need the `.replaceAll("\\s","_")` according to @chuan-wang (whitespace is ok). So better to remove this so that it's the same as the other entries in this pipeline.

Reply:
The whitespace is not allowed in HISAT2, and must be double-quoted in STAR, so I replaced the space to underscore here and used '' in STAR.

Reply:
Aha, there are tool specific differences on this? Sad times.. Ok that’s great though, thanks! I’ll just wait for the tests then merge.

Reply:
Just waiting for you to push the updates now @jun-wan :)